packages/developer-portal/src/components/pages/settings/settings-organisation-tab/organisation-form/form-schema/validation-schema.ts

```diff
@@ -37,7 +37,13 @@ export const companyInformationFormSchema = Yup.object().shape({
   [websiteField.name]: Yup.string()
     .trim()
     .required(FIELD_REQUIRED)
-    .matches(websiteRegex, websiteField.errorMessage)
+    .test({
+      message: websiteField.errorMessage,
+      test: (value) => {
+        if (!value) return true
```

Comment:
Just double checking this returns true if value is falsy.
Is this just because it gets caught by the required condition anyway?

Reply:
In the instance that there is no value, it will be caught by Yup.string().required() and display the 'required' error. In this case, I don't want to override this message by displaying the website validation message at the next step, hence return true if an empty string. Vaguely make sense? :-)

Reply:
Yes makes sense, all good then, I'll add approval.